src/app/shared/components/filters-list/full-search-bar/full-search-bar.component.ts

```
@@ -8,13 +9,13 @@ import { FormControl, Validators } from '@angular/forms';
 })
 export class FullSearchBarComponent {
   @Input() public styleClass: string;
-  public displayErrorFormControl: FormControl = new FormControl(true, Validators.requiredTrue);
+  public displayErrorFormControl: FormControl = new FormControl();
 
-  public showErrorMessage(): void {
-    this.displayErrorFormControl.setValue(false);
-  }
+  public readonly validationConstants = ValidationConstants;
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)